test/cypress/cypress/support/objects.js

```diff
@@ -0,0 +1,19 @@
+export const creds = {
+    username: 'admin',
+    email: 'admin@orchard.com',
+    password: 'Orchard1!',
+    passwordConfirmation: 'Orchard1!'
+};
+
+export const blog = {
```

Comment:
I think we could use some method like `createTestTenant` that would accept a recipe name, and return the login and prefix information in a context object. Then we could use this tenant to run a set of test for a specific feature.

This way we'll be able to run these tests in isolation, and it might create many tenants but it's not an issue.

To test if Blog or Agency work as expected, we use it specifically to check that the home page works as expected.
For other tests like Workflow, we can pick any recipe and just write what's necessary for it.

Reply:
How should I handle the prefix generation ? is a guid enough ?

Reply:
we could use an auto-incremented number as long as the state is kept such that there is no conflict. Or a timestamp value at the seconds precision + a local counter in case we create tenants in the same second.